### DISCUSSION TITLE
Code page 437 aesthetic - Alan Pope's blog

### DISCUSSION THREAD

**Discussion by giscus** [popey/popey.com-blog#27] [General]
# Code page 437 aesthetic - Alan Pope's blog

At Axiom, our design team has recently come up with a “new” (to us) 🌟aesthetic🌟 for some of our online content. It shows up in posts & ads on social media and in featured images on blog posts. Here is an example.
Announcing distributed tracing support
└ Visualize traces in a new waterfall view.
└ New dashboards to explore your services.
└ No sampling, no compromise.
└ All your traces, all the time.

https://popey.com/blog/2023/09/codepage-437-aesthetic/

**Comment by skellat** [comment]
The students I have right now have no mental framework to even conceive of this sort of thing.  The CP437 world really was quite a different time.

**Comment by thelovebug** [comment]
I used to love creating user interfaces using 437.  I knew all of the Alt-codes by heart, so I could easily build single- and double-border boxes and joins.  This form of creativity - whilst not typical for a left-brain - really fed my inner nerd.

**Reply by popey** [reply]
Have you played TIS-100 by any chance? It has the whole 437 aesthetic down so well. 
https://store.steampowered.com/app/370360/TIS100/

**Reply by thelovebug** [reply]
I haven't... gave me a flashback to Trademania though!

I'm not really a gamer, but in the same vein that you're not really a developer. 😁

**Reply by popey** [reply]
DId I tell you I found a tape a while back with Trademania on it :)

**Reply by thelovebug** [reply]
Yeah, I think you sent me a recording of the Specdrum demo at around the same time! 😆

You should try and get Trademania on a `.tap` file so that it can live on!
